File: src/Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.py
class Solution:
	# @param a list of integers
	# @return an integer
	def removeDuplicates(self, A):
		if len(A) < 2: return len(A)
		size, now = 1, A[0]
		for c in A:
			if c != now:
				A[size] = c
				size, now = size+1, c
		return size

File: src/Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.cpp
class Solution {
	public:
		int removeDuplicates(int A[], int n) {
			if(n < 2) return n;

			int size=1, now=A[0];
			for(int i = 1 ; i < n ; i++)
			{
				if(A[i] != now)
				{
					A[size] = A[i];
					size++;
					now = A[i];
				}
			}
			return size;
		}
};
